pushy/src/main/java/com/eatthepath/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -856,6 +858,19 @@ public ApnsPayloadBuilder setTimestamp(final Instant timestamp) {
         return this;
     }
 
+    /**
+     * <p>Sets the flag to wake up the app and receive a push token to send updates to the started Live Activity.</p>
+     *
+     * @return a reference to this payload builder
+     *
+     * @see <a href="https://developer.apple.com/documentation/activitykit/starting-and-updating-live-activities-with-activitykit-push-notifications">
+     *      Starting and updating Live Activities with ActivityKit push notifications</a>
+     */
+    public ApnsPayloadBuilder setInputPushToken() {
```

Comment:
We have, so far, avoided adding "one-way" switches to `ApnsPayloadBuilder`; if callers can add a thing to a payload, they ought to be able to remove it, too. My recommendation would be to model this as a boolean:

```suggestion
    public ApnsPayloadBuilder setInputPushToken(final boolean inputPushToken) {
```

…and then callers can change their minds about the property later.